osc-daemon/main.cpp

```diff
@@ -86,24 +94,26 @@ int main(int, char **) {
   // t=ModelAccuracy:ModelAccuracy:t_f;              //sets up matrix for all
   // times
   int i = 0;
+
   for (double t = 0; t < t_f; t += ModelAccuracy) {
 
     double p = 1 - ((Mo - mo * t) / Mo) * (log(Mo / (Mo - mo * t)) +
                                            1); // range function p(f) again
-    h[i] = ((g * pow2(Isp) / TWR) * p -
-                (0.5 * g * pow2(t))); // altitude as function of time
+    (*h)[i] = ((g * pow2(Isp) / TWR) * p -
+               (0.5 * g * pow2(t))); // altitude as function of time
     double V =
         g * (Isp * log(Mo / (Mo - mo * t)) + 1); // velocity as function of time
-    double h_c = h[i] + (pow2(V)) / (2 * g); // culmination alt as func of time
+    double h_c =
+        (*h)[i] + (pow2(V)) / (2 * g); // culmination alt as func of time
     if (h_c > 100000) {
       LRCT = t;
-      ASATAltAtLRCT = h[i];
+      ASATAltAtLRCT = (*h)[i];
     };
-    if (h[i] <= (SatKOE.sma - EarthRadius)) {
+    if ((*h)[i] <= (SatKOE.sma - EarthRadius)) {
       HRCT = t;
     };
 
-    outputFile << h << "," << V << "," << h_c << std::endl;
+    outputFile /* << h */ << "," << V << "," << h_c << std::endl;
```

Comment:
Not sure why you're outputing the full `h` here, this should probably be indexed by something i guess 🤷